test/it/negative.spec.js

```diff
@@ -0,0 +1,15 @@
+const compiler = require("../utils/compiler");
+const testRoots = require("../utils/tests-roots");
+const negativeRoots = testRoots.getNegativeRoots();
+
+describe("typescript - negative scenarios", () => {
+  it.each(negativeRoots)(
+    "should NOT complie successfully %s folder",
+    async rootPath => {
+      const configurationFilename = `${testRoots.NEGATIVE_ROOT_PATH}/${rootPath}/tsconfig.json`;
+      const result = compiler(configurationFilename);
+
+      expect(result.length).toBeGreaterThan(0);
```

Comment:
i am missing the reason for failure. if the reason is not what we expect, the test doesn't cover what we think it covers.
maybe we can add a ".expectedErrors.txt" file (or something similar) to each directory and compare it to the errors that the compiler returns for each test.